packages/seacas/libraries/ioss/src/visualization/catalyst/applications/ioss2catalyst/IossApplication.cxx

```diff
@@ -0,0 +1,755 @@
+// Copyright(C) 1999-2020 National Technology & Engineering Solutions
```

Comment:
Is this part of the FAODEL changes, or did it somehow get accidentally added?

Reply:
This is another part of the app integration code.  It has been removed (commit 77f7afb).  We can add it back later under a different PR.